lib/cisco_node_utils/cisco_cmn_utils.rb

```diff
@@ -367,5 +367,27 @@ def self.add_quotes(value)
         value.start_with?('"') && value.end_with?('"')
       value
     end # add_quotes
+
+    # extract value of property
+    def self.extract_value(prop, prefix=nil, match_method)
```

Comment:
If you switch the argument order of `prefix` and `match_method` you don't have to fill in the `nil` value when calling the method as follows.  Typically args with defaults should be at the end anyway.

This:
```ruby
val = Utils.extract_value('control', nil, probe_get)
```
Becomes:
```ruby
val = Utils.extract_value('control', probe_get)
```